cognite_toolkit/_cdf_tk/commands/modules.py

```diff
@@ -152,15 +159,17 @@ def _create(
 
                 if module.definition is not None and download_data:
                     for example_data in module.definition.data:
-                        if example_data.repo_type.casefold() != "github":
-                            self.warn(
-                                MediumSeverityWarning(
-                                    f"Unsupported repo type for example data: {example_data.repo_type}"
-                                )
-                            )
-                            continue
                         if example_data.repo not in downloader_by_repo:
-                            downloader_by_repo[example_data.repo] = GitHubFileDownloader(example_data.repo)
+                            try:
+                                downloader_cls = _FILE_DOWNLOADERS_BY_TYPE[example_data.repo_type]
+                                downloader_by_repo[example_data.repo] = downloader_cls(example_data.repo)
```

Comment:
Should be moved outside try except. I assume the Keyerror is only intended for the 164 line